packages/react-scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-scripts",
-  "version": "1.0.11",
+  "name": "@connected-home/react-scripts",
+  "version": "1.0.11.1",
```

Comment:
I was just trying to create a new version number so I could publish to npm. Ideally I would just leave the version number alone but I couldn't figure out how to overwrite an npm publish.